docs/changelog.md

```
@@ -1,6 +1,8 @@
 # Changelog
 
-### Unreleased
+### 0.5.0
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)